inst/WORDLIST

```diff
@@ -4,3 +4,9 @@ Sabanés
 funder
 geoms
 ggplot
+doi
+EnvStats
+GGAlly
```

Comment:
```suggestion
GGally
```

I think that it is `GGally` https://www.rdocumentation.org/packages/GGally/versions/1.5.0 or are you referring to another package ?